config/config.toml.example

```diff
@@ -74,6 +74,9 @@ split-region-max-num = 1000
 # In order to support "drop primary key" operation , this flag must be true and the table does not have the pkIsHandle flag.
 alter-primary-key = false
 
+# serverVersion is used to show the version of tidb when log in.
```

Comment:
Please add the comment to explain the version when `server-version = ""`(Original TiDB version)